src/test/resources/mockData/test_instance_to_post.json

```diff
@@ -0,0 +1,28 @@
+{
+  "id": "c5d662af-b0be-4851-bb9c-de70bba3dfce",
+  "source": "Local: MARC",
+  "title": "ADVANCING LIBRARY EDUCATION: TECHNOLOGICAL INNOVATION AND INSTRUCTIONAL DESIGN",
+  "contributors": [
+    {
+      "name": "Sigal, Ari",
+      "contributorNameTypeId": "2b94c631-fca9-4892-a730-03ee529ffe2a",
+      "primary": true
+    }
+  ],
+  "identifiers": [
+    {
+      "identifierTypeId": "2e48e713-17f3-4c13-a9f8-23845bb210af",
+      "value": "9781466636897"
+    },
+    {
+      "identifierTypeId": "6051f95c-028e-4c6a-8a9e-ee689dd51453",
```

Comment:
Don't you need a mock for  identifier type?

Reply:
I think we should add it when we will need it